package.json

```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
-    "start": "tsc && node ./dist/src/app.js",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
+    "start": "tsc && /Users/victorryabkov/.nvm/versions/node/v16.20.2/bin/node ./dist/src/app.js",
```

Comment:
remove /Users/victorryabkov/.nvm/versions/node/v16.20.2/bin/node

Reply:
Forgot about it. Thank you!

Reply:
Done